scripts/bash/enceladus_env.template.sh

```diff
@@ -34,12 +34,14 @@ STD_DEFAULT_DRIVER_MEMORY=""
 STD_DEFAULT_DRIVER_CORES=""
 STD_DEFAULT_EXECUTOR_MEMORY=""
 STD_DEFAULT_EXECUTOR_CORES=""
+STD_DEFAULT_DRA_EXECUTOR_MEMORY=""
+STD_DEFAULT_DRA_EXECUTOR_CORES=""
 # setting num executors disables DRA
 STD_DEFAULT_NUM_EXECUTORS=""
 
 # Dynamic Resource Allocation
 # also enables external shuffle service and adaptive execution for consistent setup
-STD_DEFAULT_DRA_ENABLED=false
+STD_DEFAULT_DRA_ENABLED=true
```

Comment:
**To consider:** If we will have a switch `--disable-dra` and no `--enable-dra` that this doesn't have much sense IMO.
On the other hand, I think it would be good to have both switches and then this default makes sense.

(Obviously same applies to `CONF_DEFAULT_DRA_ENABLED`)

Reply:
Having switch `--set-dra` makes this comment moot.